3rdparty/debian/control

```diff
@@ -1,10 +1,14 @@
-Package: rapiddisk
-Version: 8.1.0-1
```

Comment:
Well, I am drawing my own chart these days too, to be honest. I did this before but I am not a guru at all.
Anyway, the `debian/control` file is not supposed to provide the package version, it is a generic description of the package (altough it may change upon new releases, for example if new dependencies are needed), as stated [here:](https://www.debian.org/doc/debian-policy/ch-controlfields.html#source-package-control-files-debian-control)

> The debian/control file contains the most vital **(and version-independent)** information about the source package and about the binary packages it creates.

The version number of the package is obtained from the topmost entry in the `debian/changelog` file ([docs](https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog)).

BTW, If you wonder why I moved the `debian` directory in the `3rdparty` subdir, that's because, long story short, a `rapiddisk` release (aka the source in the tarball downloadable from github from the tag page) should not be "debianized" by default.